library/alloc/src/alloc.rs

```diff
@@ -125,7 +125,7 @@ pub unsafe fn dealloc(ptr: *mut u8, layout: Layout) {
 /// of the allocator registered with the `#[global_allocator]` attribute
 /// if there is one, or the `std` crate’s default.
 ///
-/// This function is expected to be deprecated in favor of the `realloc` method
+/// This function is expected to be deprecated in favor of the `grow` method
```

Comment:
Hm, this actually seems wrong? `realloc` can grow and shrink...

Reply:
@Amanieu do you perhaps know the context here? It seems like we can't say this is 1:1 with Allocator trait methods. Maybe we should document this as (future) deprecated in favor of `grow` or `shrink` depending on what the caller wants?

Reply:
Yes that's correct: `realloc` is deprecated in favor of the `grow` and `shrink` methods.